.circleci/config.yml

```diff
@@ -243,26 +214,16 @@ workflows:
               only: /^v.*/
             branches:
               ignore: /.*/
-      - sign_release:
-          context: cloudskiff-signing
-          requires:
-            - build
-          filters:
-            tags:
-              only: /^v.*/
-            branches:
-              ignore: /.*/
       - release:
           context: driftctl
```

Comment:
We also need the `cloudskiff-signing` context here where remain the private key for signature